puppet-agent-installer.sh

```diff
@@ -61,6 +61,25 @@ detect_rhel_or_oracle_5 ( ) {
 
 }
 
+detect_ubuntu_1804 ( ) {
+
+  if grep -E 'DISTRIB_RELEASE=18.04' /etc/lsb-release &> /dev/null; then
+    cd /tmp
+    if [[ ${PUPPET_5} -eq 0 ]] ; then
+      curl -s -O http://apt.puppet.com/puppet5-release-bionic.deb
+      dpkg -i puppet5-release-bionic.deb
+      rm -f puppet5-release-bionic.deb
+    else
+      curl -s -O http://apt.puppet.com/puppetlabs-release-pc1-bionic.deb
```

Comment:
Ubuntu 18.04 Bionic will have no Puppet 4 packages, so the `else` block is not useful.